Marlin/language_en.h

```diff
@@ -508,5 +508,8 @@
 #ifndef MSG_DELTA_CALIBRATE_CENTER
   #define MSG_DELTA_CALIBRATE_CENTER          "Calibrate Center"
 #endif
-
+#define MSG_CHANGE_FILAMENT                 "Filament"
+#define MSG_REMOVE_FILAMENT                 "Remove"
+#define MSG_INSERT_FILAMENT                 "Insert"
+#define MSG_INSERT_FILAMENT_SLOW            "Slow Insert"
 #endif // LANGUAGE_EN_H
```

Comment:
Yes, no warnings as you can see  ![here](http://codersquirrel.github.io/trash/Screenshot%20from%202016-04-24%2019:25:13.png)